server/testdriver/core/src/main/java/org/infinispan/server/test/core/rollingupgrade/RollingUpgradeConfigurationBuilder.java

```diff
@@ -0,0 +1,107 @@
+package org.infinispan.server.test.core.rollingupgrade;
+
+import java.net.SocketAddress;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.commons.configuration.StringConfiguration;
+
+public class RollingUpgradeConfigurationBuilder {
+   private final String fromVersion;
+   private final String toVersion;
+
+   private int nodeCount = 3;
+   private boolean xSite;
+   private String jgroupsProtocol = "tcp";
+   private int serverCheckTimeSecs = 30;
+   private boolean useSharedDataMount = true;
+   private BiConsumer<Throwable, RollingUpgradeHandler> exceptionHandler = (t, uh) -> {
+      t.printStackTrace();
+   };
+
+   private Consumer<String> logConsumer = System.out::println;
```

Comment:
I guess this is to keep it simple and avoid bringing dependencies into scripts (e.g., JBang), but should this utilize a logger instead? I was wondering if we'll have problems in our tests, and the output wasn't written in the file.